_jobs/software-engineer.md

```diff
@@ -21,41 +23,47 @@ We are looking for a Software Engineer to support a range of new and ongoing cli
 - Document your work, including asynchronous communications, to inform internal and client teams
 - Coordinate with product, design, and engineering to deliver delightful products and services
```

Comment:
```suggestion
- Coordinate with product, design, and engineering to deliver delightful products and services that meet current Web Content Accessibility Guidelines (WCAG)
```

Reply:
I think I'd prefer more objective adjectives like `performant, secure, well-designed` rather than `delightful` - but maybe doesn't matter that much. Definitely want WCAG in there though!